src/KubeConfigModels/UserCredentials.cs

```diff
@@ -8,9 +9,15 @@ public class UserCredentials
         [YamlMember(Alias = "client-certificate-data")]
         public string ClientCertificateData { get; set; }
 
+        [YamlMember(Alias = "client-certificate")]
+        public string ClientCertificate { get; set; }

```

Comment:
Need to review unit tests. Thought that client-certifidate-data hold the route to a X509 certificate, but concluded now it is the certificate itself. Additional tests should be written to cover the client-certificate scenario.